### PR TITLE
Prevent false positive on duplicate rows from `translations.txt`

### PR DESCRIPTION
**Pull request by lionel-nj**
closes #492 

**Summary:**

This PR provides support to prevent gtfs-validator from generating false positive when checking for duplicate entity in table `translations.txt`.

**Expected behavior:** 

The sample of incorrect validation errors:

> table_name | field_name | language | translation | record_id
> stops | stop_name | sv | Hintza Knaps gata | 1460
> stops | stop_name | sv | Hintza Knaps gata | 1461
> stops | stop_name | sv | Hintza Knaps gata | 835
(https://github.com/MobilityData/gtfs-validator/issues/492#issue-749505217) 

This sample should not generate `DuplicateEntityNotice`

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Run the unit tests with `gradle test` to make sure you didn't break anything
- [ ] Format the title like "Fix #<issue_number> - <short description of fix and changes>" (for example - "Fix #1111 - Check for null value before using field")
- [ ] Linked all relevant issues
- [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

**Comment by lionel-nj** [comment]
Hello @gabrbuiv ! 
First of all thank you for your contribution! 

1. We added FÖLI's dataset to our end_to_end workflow, you'll find the produced artifacts [here](https://github.com/MobilityData/gtfs-validator/suites/1554270338/artifacts/27813977)
   - [workflow execution log](https://github.com/MobilityData/gtfs-validator/runs/1448581753?check_suite_focus=true)
2. The process will be as follows:
   1. We will shortly add tests (that will fail in a first time): the goal is to make sure that the bug you encountered is handled properly
   2. Once everyone one agrees on the tests, we will proceed to code implementation
   

@fabrice-v

**Comment by fabrice-v** [comment]
@lionel-nj Could you also modify the end to end and activate it in this present PR so we could all see here the shape of the issue?

**Comment by fabrice-v** [comment]
(that should have been the normal flow before the upstream end to end is modified, first in the forked PR :) )

File: .github/workflows/end_to_end.yml [hunk 1]
```diff
@@ -3,9 +3,7 @@ name: End to end
 on:
   push:
     branches: [ master, fix-false-positive-duplicate-translations ] #<-- replace transport-agency-name by the name of the agency/publisher
-  pull_request_target:
-    branches: [ fix-false-positive-duplicate-translations ]
-
+
```
Comment (fabrice-v) [inline .github/workflows/end_to_end.yml:6]: 👀

**Comment by gabrbuiv** [comment]
@lionel-nj @fabrice-v thank you for your prompt reaction! Let me know if you need any further information from me.

**Comment by lionel-nj** [comment]
@gabrbuiv It seems that GitHub Actions are not automatically ran on a fork 
(https://github.community/t/how-to-run-and-enable-github-actions-on-forked-repo-with-github-api/17232). Could you try enabling them so that we could determine if further investigation is required to run them or not please?

8a28c79db26e3c2d82f2d0694ae0762b2ef7763b

**Comment by gabrbuiv** [comment]
@lionel-nj I'm not very familiar with Actions, but it seems that I've enabled them. Let me know if it won't work.

**Comment by lionel-nj** [comment]
Thank you, @gabrbuiv. The actions did not execute. We will investigate this.

File: .github/workflows/end_to_end.yml (hunk 1 above)
Comment (lionel-nj) [reply .github/workflows/end_to_end.yml]: https://github.com/gabrbuiv/gtfs-validator/pull/1/commits/5efa79e94ad6919d098f697a4d6550d0fc3c093c executed the end_to_end workflow on this PR

**Comment by lionel-nj** [comment]
FYI, the `end_to_end` workflow was successfully executed: https://github.com/gabrbuiv/gtfs-validator/actions/runs/385408730. On this same page you will find all validated datasets and their related validation reports (including FÖLI's). 
@gabrbuiv

**Pull request closed without merging**